shared/attributes.asciidoc

```diff
@@ -55,6 +55,7 @@ endif::[]
 :apm-go-ref:           https://www.elastic.co/guide/en/apm/agent/go/current
 :apm-dotnet-ref:       https://www.elastic.co/guide/en/apm/agent/dotnet/current
 :apm-php-ref:          https://www.elastic.co/guide/en/apm/agent/php/current
+:apm-ios-ref:          https://www.elastic.co/guide/en/apm/agent/ios/current
```

Comment:
These attributes are used for cross-doc linking. For example, when linking from the stack or another agent to the ios docs, instead of copy/pasting the URL, you'd use this attribute + the id.

```
{apm-ios-ref}/setup.html[link text]
```

This enables link checking in our docs.

Reply:
We define `prefix` as `swift` in `conf.yaml. That means this should be: 
```suggestion
:apm-ios-ref:          https://www.elastic.co/guide/en/apm/agent/swift/current
```
Unless you want to make it `ios` everwhere.

Reply:
I think swift is more appropriate

Reply:
Good catch